cli/src/main/java/org/apache/falcon/cli/FalconExtensionCLI.java

```diff
@@ -51,6 +51,7 @@
 
     // Extension commands
     private static final String ENUMERATE_OPT = "enumerate";
+    private static final String JOB_LIST_OPT = "jobList";
```

Comment:
correctly said so, have revert my changes and have modified the existing list API.